src/webhooks/registry.ts

```diff
@@ -115,7 +115,7 @@ function isSuccess(
 
 // 2020-07 onwards
 function versionSupportsEndpointField() {
-  return ShopifyUtilities.versionCompatible(ApiVersion.July20);
+  return ShopifyUtilities.versionCompatible('2020-07' as unknown as ApiVersion);
```

Comment:
Since we're still using `July20` in our logic what was the intent of removing the following from our `ApiVersion` enum?

* `July20`
* `October20`
* `January21`

I would think it would be better to leave the original enum values, rather than using this double-cast trick to get around the type checker.

Reply:
That's a fair point - I think we should actually just fully drop this function - `2020-07` is no longer supported, so this branch should never run in a real world setting.